src/DependencyInjection/DocusignExtension.php

```diff
@@ -75,12 +76,29 @@ public function load(array $configs, ContainerBuilder $container): void
             ->setPublic(false)
             ->addTag('routing.loader');
 
+        $clickwrapExtensionDefinition = $container->register('docusign.twig.extension.clickwrap', ClickwrapExtension::class)
```

Comment:
shouldn't we check that twig is loaded? So it's not loaded for nothing.

Reply:
I suppose a simple class_exist() over Twig_Environment should do.

Reply:
I don't think it's useful. This is just a simple service, with a tag. If the user uses twig, the tag will be used. Otherwise, the service is available but useless, which is not an error